rpc/types/src/bin.rs

```diff
@@ -159,10 +161,10 @@ impl Default for GetBlocksResponse {
     }
 }
 
-/// Data within [`GetBlocksResponse::PoolInfoNone`].
+/// Common data within all of [`GetBlocksResponse`]'s variants.
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 #[derive(Clone, Default, Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
-pub struct GetBlocksResponsePoolInfoNone {
+pub struct GetBlocksResponseHeader {
```

Comment:
### Misc
Added a flattened `GetBlocksResponseHeader` type that contains common fields from all `GetBlocksResponse` variants.